src/Components/ProjectPage/ProjectCreationForm.js

```diff
@@ -103,17 +104,25 @@ export default function ProjectCreationForm(props) {
   };
 
   useEffect(() => {
-    let token = localStorage.getItem('token');
-    axios
-      .get('https://secret-sauce.azurewebsites.net/auth/company/', {
-        headers: {
-          'Content-Type': 'application/json',
-          Accept: 'application/json',
-          Authorization: `Token ${token}`
-        }
-      })
-      .then(res => setOrganizationList(res.data));
-  }, []);
+    // CHECK FOR USER COMPANY, BETTER CHANGE THE HARDCODE COMPANY ID TO STATIC FILE
+    if (userCompany === '8bead4d0-fb36-4c05-8fbb-825feed1ba4f') {
```

Comment:
Should check for whether user is_staff instead of checking user company

Reply:
this has been fixed

Reply:
Remember to push. Dun see the commit yet